src/deb/RuneLiteNoHwAcc.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Encoding=UTF-8
+Name=RuneLite (disabled hardware acceleration)
+Comment=OldSchool RuneScape client. Launches with disabled hardware acceleration.
+Terminal=false
+Type=Application
+Categories=Game
+StartupNotify=true
+Exec=/usr/games/runelite-no-hw-acc
```

Comment:
This should just call the normal executable with extra arguments

Reply:
I did it this way just so it can be easily launched from command line with tab completion (although not that this would be important really)